x/poe/types/msg.go

```diff
@@ -81,7 +81,7 @@ func (msg MsgCreateValidator) ValidateBasic() error {
 	}
 
 	if !msg.Value.IsValid() || !msg.Value.Amount.IsPositive() {
-		return stakingtypes.ErrBadDelegationAmount
```

Comment:
those staking errors are gone?

Reply:
This one is gone. Others still exists.  In the staking/errors:
> // TODO: Many of these errors are redundant. They should be removed and replaced
// by sdkerrors.ErrInvalidRequest.